applications/nrf5340_audio/src/audio/audio_datapath.c

```diff
@@ -898,29 +899,32 @@ void audio_datapath_pres_delay_us_get(uint32_t *delay_us)
 	*delay_us = ctrl_blk.pres_comp.pres_delay_us;
 }
 
-void audio_datapath_stream_out(const uint8_t *buf, size_t size, uint32_t sdu_ref_us, bool bad_frame,
-			       uint32_t recv_frame_ts_us)
+void audio_datapath_stream_out(struct audio_data *audio_frame)
 {
 	if (!ctrl_blk.stream_started) {
 		LOG_WRN("Stream not started");
 		return;
 	}
 
 	/*** Check incoming data ***/
+	struct net_buf *buf = audio_frame->data;
```

Comment:
Since this is audio data, suggest we specify a bit more than `buf`.
What about "audio_buf", "net_buf_audio", "net_buf_frame" ?

Reply:
audio_buf used